code/controllers/configuration.dm

```diff
@@ -641,31 +641,31 @@
 					config.max_maint_drones = text2num(value)
 
 				if("expected_round_length")
-					config.expected_round_length = MinutesToTicks(text2num(value))
+					config.expected_round_length = text2num(value) MINUTES
```

Comment:
`MINUTES` is only `*10`, no tick value is used. You sure you werent meant to use these?

```dm
#define DS2TICKS(DS) ((DS)/world.tick_lag)
#define TICKS2DS(T) ((T) TICKS)
```

Reply:
It is used in the context of `world.time`.
I also do not want to change functionality in this PR unless really needed.

Reply:
Oh thats fine then. Reading this code now makes me realise how much utter garbo is here.